Week-2/Assignment-1/style.css

```diff
@@ -0,0 +1,115 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap');
+
+html,
+body {
+    padding: 5px;
```

Comment:
Just use multiples of 4px

Reply:
Used multiples of 4px in padding, margin,width, height etc